### ISSUE TITLE
Mod Matrix

### ISSUE THREAD

**Issue by bhollis** [DestinyItemManager/DIM#9828]
@vivekhnz proposed this neat way of presenting mods that have a value for every damage type:

![](https://user-images.githubusercontent.com/17512262/231764660-5be5e4d8-5fec-41ba-86b4-b4c782a1a972.png)

This could theoretically make the mod picker a lot more compact.

cc @ryan-rushton

**Comment by sidferreira** [comment]
@bhollis is it in `src/app/loadout/ModPicker.tsx`?

**Comment by bhollis** [comment]
That's where it would go, yeah

**Comment by ryan-rushton** [comment]
Yeah that is much more compact. It removes the upfront information about mods and such but I have long suspected that most of the users of this are power users and can identify most mods via icon sight anyways. If they can somehow access more detailed information through a CTA that would be ideal.
